en/README.md

```diff
@@ -11,4 +11,23 @@ Because MAVLink doesn't require any additional framing it is very well suited fo
 
 MAVLink was first released early 2009 by Lorenz Meier and has now a [significant number of contributors](https://github.com/mavlink/mavlink/graphs/contributors).
 
-The content of this book is CC-BY 4.0 licensed.
+## Forums and Chat {#support}
+
+The core development team and community are active on the following forums and chat channels:
+
+* [PX4 Discuss](http://discuss.px4.io/) (*recommended*)
+* [Slack](http://slack.px4.io) (sign up) - #mavlink channel
```

Comment:
What are the right channels?
- AFAIK it is really just the slack #mavlink channel? 
- Should we point people to discussion boards, and if so should we create a MAVLINK channel. My leaning is "yes" but perhaps just point them to software dev channel?
- Is the old mailing list still used?: [Google Groups](http://groups.google.com/group/mavlink)
- Should MAVLink developers be pointed to/attend the weekly devcall?
- Should we provide links to the ardupilot area for mavlink discussions?

Reply:
> AFAIK it is really just the slack #mavlink channel?

I think so.

> Honestly, I'm overwhelmed having to keep track of forum, slack, and github. That's a separate problem but given that I don't know the correct answer.

Reply:
Thanks @julianoes - I think it is more important to have correct info and a monitored channel than every channel under the sun. I'm going to just have slack and then merge this. Will deal with/discuss the devcall question separately. Thanks.